go.mod

```diff
@@ -38,9 +38,11 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 	sigs.k8s.io/yaml v1.3.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v1.6.0-alpha.0.0.20230214095022-ad92502b1a57
+	volcano.sh/apis v0.0.0-20230812071333-904c042dbf63
 )
 
+replace volcano.sh/apis v0.0.0-20230812071333-904c042dbf63 => github.com/lowang-bh/apis v0.0.0-20230812071333-904c042dbf63
+
```

Comment:
this is not good.

Reply:
This is a temporary commit until apis's relative pr [feat: add podgroup uninqueueable condition type and un-enqueueable reason #113](https://github.com/volcano-sh/apis/pull/113) merged.

Reply:
may be you should make this PR to WIP status